### PR TITLE
[gui-tests] Refactor sync paths for oc10 and ocis

### PR DESCRIPTION
**Pull request by saw-jan**
Sync paths between oc10 and ocis are different. When specifying `~/Alice/` as sync path, files are synced to that path in oc10 but in ocis, files are synced to the `~/Alice/Personal` folder.
In order to share steps while creating spaces related tests, these refactor are necessary. And, it is recommended to use `getResourcePath` function to get the necessary real file/folder path.

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>GUI-tests-ocis</strong> https://drone.owncloud.com/owncloud/client/14011/7/1

:boom: The GUI tests failed.

GUI Logs: https://cache.owncloud.com/public/owncloud/client/14011/ocis/guiReportUpload/index.html

<details>

Server Logs: https://cache.owncloud.com/public/owncloud/client/14011/ocis/guiReportUpload/serverlog.log

Screenshots:
- [Adding_account_with_vfs_enabled.png](https://cache.owncloud.com/public/owncloud/client/14011/ocis/guiReportUpload/screenshots/Adding_account_with_vfs_enabled.png)
- [File_with_long_name_can_be_synced.png](https://cache.owncloud.com/public/owncloud/client/14011/ocis/guiReportUpload/screenshots/File_with_long_name_can_be_synced.png)
- [File_with_spaces_in_the_name_can_sync.png](https://cache.owncloud.com/public/owncloud/client/14011/ocis/guiReportUpload/screenshots/File_with_spaces_in_the_name_can_sync.png)
- [Filenames_that_are_rejected_by_the_server_are_reported.png](https://cache.owncloud.com/public/owncloud/client/14011/ocis/guiReportUpload/screenshots/Filenames_that_are_rejected_by_the_server_are_reported.png)
- [Many_subfolders_can_be_synced.png](https://cache.owncloud.com/public/owncloud/client/14011/ocis/guiReportUpload/screenshots/Many_subfolders_can_be_synced.png)
- [Syncing_a_file_to_the_server.png](https://cache.owncloud.com/public/owncloud/client/14011/ocis/guiReportUpload/screenshots/Syncing_a_file_to_the_server.png)
- [Syncing_folders_each_having_500_files.png](https://cache.owncloud.com/public/owncloud/client/14011/ocis/guiReportUpload/screenshots/Syncing_folders_each_having_500_files.png)
- [Try_to_enable_experimental_vfs_option_and_cancel_it.png](https://cache.owncloud.com/public/owncloud/client/14011/ocis/guiReportUpload/screenshots/Try_to_enable_experimental_vfs_option_and_cancel_it.png)
- [various_types_of_files_can_be_synced_from_client_to_server.png](https://cache.owncloud.com/public/owncloud/client/14011/ocis/guiReportUpload/screenshots/various_types_of_files_can_be_synced_from_client_to_server.png)

</details>

**Comment by ownclouders** [comment]
Results for <strong>GUI-tests-oc10</strong> https://drone.owncloud.com/owncloud/client/14011/6/1

:boom: The GUI tests failed.

GUI Logs: https://cache.owncloud.com/public/owncloud/client/14011/oc10/guiReportUpload/index.html

<details>

Server Logs: https://cache.owncloud.com/public/owncloud/client/14011/oc10/guiReportUpload/serverlog.log

Screenshots:
- [sharee_edits_shared_files_and_again_try_to_edit_after_write_permission_is_revoked.png](https://cache.owncloud.com/public/owncloud/client/14011/oc10/guiReportUpload/screenshots/sharee_edits_shared_files_and_again_try_to_edit_after_write_permission_is_revoked.png)
- [simple_sharing_of_a_file_by_public_link_with_default_expiration_date.png](https://cache.owncloud.com/public/owncloud/client/14011/oc10/guiReportUpload/screenshots/simple_sharing_of_a_file_by_public_link_with_default_expiration_date.png)
- [simple_sharing_of_a_file_by_public_link_with_password_and_expiration_date.png](https://cache.owncloud.com/public/owncloud/client/14011/oc10/guiReportUpload/screenshots/simple_sharing_of_a_file_by_public_link_with_password_and_expiration_date.png)
- [simple_sharing_of_file_and_folder_by_public_link_with_expiration_date.png](https://cache.owncloud.com/public/owncloud/client/14011/oc10/guiReportUpload/screenshots/simple_sharing_of_file_and_folder_by_public_link_with_expiration_date.png)
- [simple_sharing_of_file_and_folder_by_public_link_with_password.png](https://cache.owncloud.com/public/owncloud/client/14011/oc10/guiReportUpload/screenshots/simple_sharing_of_file_and_folder_by_public_link_with_password.png)
- [simple_sharing_of_file_and_folder_by_public_link_without_password.png](https://cache.owncloud.com/public/owncloud/client/14011/oc10/guiReportUpload/screenshots/simple_sharing_of_file_and_folder_by_public_link_without_password.png)

</details>

**Comment by ownclouders** [comment]
Results for <strong>GUI-tests-ocis</strong> https://drone.owncloud.com/owncloud/client/14015/7/1

:boom: The GUI tests failed.

GUI Logs: https://cache.owncloud.com/public/owncloud/client/14015/ocis/guiReportUpload/index.html

<details>

Server Logs: https://cache.owncloud.com/public/owncloud/client/14015/ocis/guiReportUpload/serverlog.log

Screenshots:
- [Filenames_that_are_rejected_by_the_server_are_reported.png](https://cache.owncloud.com/public/owncloud/client/14015/ocis/guiReportUpload/screenshots/Filenames_that_are_rejected_by_the_server_are_reported.png)
- [Many_subfolders_can_be_synced.png](https://cache.owncloud.com/public/owncloud/client/14015/ocis/guiReportUpload/screenshots/Many_subfolders_can_be_synced.png)
- [Verify_one_empty_folder_with_a_length_longer_than_the_allowed_limit_will_not_be_synced.png](https://cache.owncloud.com/public/owncloud/client/14015/ocis/guiReportUpload/screenshots/Verify_one_empty_folder_with_a_length_longer_than_the_allowed_limit_will_not_be_synced.png)
- [Verify_pre_existing_folders_in_local_(Desktop_client)_are_copied_over_to_the_server.png](https://cache.owncloud.com/public/owncloud/client/14015/ocis/guiReportUpload/screenshots/Verify_pre_existing_folders_in_local_(Desktop_client)_are_copied_over_to_the_server.png)
- [Verify_that_you_can_create_a_subfolder_with_long_name.png](https://cache.owncloud.com/public/owncloud/client/14015/ocis/guiReportUpload/screenshots/Verify_that_you_can_create_a_subfolder_with_long_name.png)
- [various_types_of_files_can_be_synced_from_server_to_client.png](https://cache.owncloud.com/public/owncloud/client/14015/ocis/guiReportUpload/screenshots/various_types_of_files_can_be_synced_from_server_to_client.png)

</details>

**Comment by ownclouders** [comment]
Results for <strong>GUI-tests-oc10</strong> https://drone.owncloud.com/owncloud/client/14015/6/1

:boom: The GUI tests failed.

GUI Logs: https://cache.owncloud.com/public/owncloud/client/14015/oc10/guiReportUpload/index.html

<details>

Server Logs: https://cache.owncloud.com/public/owncloud/client/14015/oc10/guiReportUpload/serverlog.log

Screenshots:
- [Filenames_that_are_rejected_by_the_server_are_reported.png](https://cache.owncloud.com/public/owncloud/client/14015/oc10/guiReportUpload/screenshots/Filenames_that_are_rejected_by_the_server_are_reported.png)
- [Many_subfolders_can_be_synced.png](https://cache.owncloud.com/public/owncloud/client/14015/oc10/guiReportUpload/screenshots/Many_subfolders_can_be_synced.png)
- [Verify_one_empty_folder_with_a_length_longer_than_the_allowed_limit_will_not_be_synced.png](https://cache.owncloud.com/public/owncloud/client/14015/oc10/guiReportUpload/screenshots/Verify_one_empty_folder_with_a_length_longer_than_the_allowed_limit_will_not_be_synced.png)
- [Verify_pre_existing_folders_in_local_(Desktop_client)_are_copied_over_to_the_server.png](https://cache.owncloud.com/public/owncloud/client/14015/oc10/guiReportUpload/screenshots/Verify_pre_existing_folders_in_local_(Desktop_client)_are_copied_over_to_the_server.png)
- [Verify_that_you_can_create_a_subfolder_with_long_name.png](https://cache.owncloud.com/public/owncloud/client/14015/oc10/guiReportUpload/screenshots/Verify_that_you_can_create_a_subfolder_with_long_name.png)
- [sharee_creates_a_file_and_a_folder_inside_a_shared_folder.png](https://cache.owncloud.com/public/owncloud/client/14015/oc10/guiReportUpload/screenshots/sharee_creates_a_file_and_a_folder_inside_a_shared_folder.png)
- [sharee_edits_content_of_files_shared_by_sharer.png](https://cache.owncloud.com/public/owncloud/client/14015/oc10/guiReportUpload/screenshots/sharee_edits_content_of_files_shared_by_sharer.png)
- [sharee_tries_to_create_a_file_and_a_folder_inside_a_shared_folder_without_write_permission.png](https://cache.owncloud.com/public/owncloud/client/14015/oc10/guiReportUpload/screenshots/sharee_tries_to_create_a_file_and_a_folder_inside_a_shared_folder_without_write_permission.png)
- [various_types_of_files_can_be_synced_from_server_to_client.png](https://cache.owncloud.com/public/owncloud/client/14015/oc10/guiReportUpload/screenshots/various_types_of_files_can_be_synced_from_server_to_client.png)

</details>

File: test/gui/suite.conf [hunk 1]
```diff
@@ -4,4 +4,5 @@ HOOK_SUB_PROCESSES=false
 IMPLICITAUTSTART=false
 LANGUAGE=Python
 OBJECTMAPSTYLE=script
+VERSION=3
```
Comment (saw-jan) [inline test/gui/suite.conf:7]: This is needed for Squish IDE to generate UI selectors as an object. Previously, it got removed in one of the refactoring PRs.

**Review by individual-it — APPROVED** [review]

**Pull request merged**